composite-actions/enforce-jira-issue-key/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Ensure JIRA issue is linked
+description: Check if JIRA issue key is contained in branch name or PR title
+
+inputs:
+  jira_base_url:
+    description: URL of Jira instance
+    required: true
+  jira_user_email:
+    description: Access Token for Authorization (see https://github.com/atlassian/gajira-login#enviroment-variables for details)
+    required: true
+  jira_api_token:
+    description: email of the user for which Access Token was created for
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Log into Jira
+      uses: atlassian/gajira-login@v2.0.0
+      env: 
+        JIRA_BASE_URL: ${{ inputs.jira_base_url }}
+        JIRA_USER_EMAIL: ${{ inputs.jira_user_email }}
+        JIRA_API_TOKEN: ${{ inputs.jira_api_token }}
+    - name: Find issue key in PR title
+      id: issue_key_pr_title
+      continue-on-error: true
+      uses: atlassian/gajira-find-issue-key@v2.0.2
+      with:
+        string: ${{ github.event.pull_request.title }}
+        from: "" # required workaround for bug https://github.com/atlassian/gajira-find-issue-key/issues/24
+    - name: Find issue key in branch name
+      continue-on-error: true
+      id: issue_key_branch_name
+      uses: atlassian/gajira-find-issue-key@v2.0.2
+      with:
+        string: ${{ github.event.pull_request.head.ref }}
+        from: "" # required workaround for bug https://github.com/atlassian/gajira-find-issue-key/issues/24
+    - name: Check if issue key was found
```

Comment:
You need to set the `shell:` keyword here.

Reply:
@brennerm Yes, just found that through debugging, thanks
Could you check it once more? I've tested it in https://github.com/Telecominfraproject/wlan-cloud-ucentralsec/pull/27